<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Traffic lights</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="StartStopBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StepBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>25</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="CustomOpenGLWidget" name="trafficview">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>540</width>
       <height>540</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="StatBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>180</width>
      <height>100</height>
     </rect>
    </property>
    <widget class="QLabel" name="WaitTimeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>100</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Total wait time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="QueuedLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>100</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Total queued:</string>
     </property>
    </widget>
    <widget class="QLabel" name="WaitTime">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>60</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="QueuedTotal">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>60</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="StepLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Step:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Step">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>60</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>customopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
